### PR TITLE
[INLONG-904][DOC] Add MySQL to Iceberg example document for data sync

### LINKED ISSUE #904: [Improve][Doc] Add MySQL to Iceberg example document for data sync
### Description

Add `MySQL` to the `Iceberg` example document for data sync.

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by haifxu**
### Prepare a Pull Request

- Fixes #904 

### Motivation

Add MySQL to Iceberg example document for data sync

### Modifications

Add MySQL to Iceberg example document for data sync

### Documentation
Chinese：
mysql_iceberg_example_zh

English：
mysql_iceberg_example

### CODE REVIEW

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/quick_start/data_sync/mysql_iceberg_example.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+![Create Stream_Source](img/mysql_iceberg/create_stream_source.png)
+
+:::note
+- 这里读取模式选择了 `全量+增量`，表中的存量数据也会被采集，`仅增量` 模式则不会。
+- 表名白名单格式为 `<dbName>.<tableName>`，支持正则表达。
+:::
+
+### 创建数据目标
+数据目标中点击 [新建] → [Iceberg]，设置数据目标名称并选择创建好的 Iceberg 数据节点,
+库表名称我们可以选择与数据源一直，或者自定义。
```
Comment (EMsnap) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/quick_start/data_sync/mysql_iceberg_example.md:64]: 一致

Comment (haifxu) [reply i18n/zh-CN/docusaurus-plugin-content-docs/current/quick_start/data_sync/mysql_iceberg_example.md]: Done.

File: docs/quick_start/data_sync/mysql_iceberg_example.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+![Create Stream_Source](img/mysql_iceberg/create_stream_source.png)
+
+:::note
+- If the read mode is `Full amount + Incremental`, the existing data in the source table will also be collected, but the `Incremental` mode will not.
+- The table white list format is `<dbName>.<tableName>` and supports regular expressions.
+:::
+
+### Create Data Sink
+In the data sink, click [New] → [Iceberg] to configure the sink name and created Iceberg data node.
+We can choose the data sink to have the same database table name as the data source, or customize it.
```
Comment (EMsnap) [inline docs/quick_start/data_sync/mysql_iceberg_example.md:66]: please add a description on the customize rule

**Review by EMsnap — APPROVED** [review]
LGTM

**Review by dockerzhang — APPROVED** [review]

**Pull request merged**